arviz/utils/xarray_utils.py

```diff
@@ -509,11 +509,19 @@ def infer_dtypes(self):
             r'//.*?$|/\*.*?\*/|\'(?:\\.|[^\\\'])*\'|"(?:\\.|[^\\"])*"',
             re.DOTALL|re.MULTILINE
         )
+        STAN_INTEGER = r"int"
```

Comment:
i like this change, but the linter is relentless! these need to be lower snakecase (`stan_limits`, `stan_shape`, etc), and line 524 needs to be broken up (I'd probably do it carefully in a few steps).